planning/behavior_velocity_planner/src/scene_module/blind_spot/scene.cpp

```diff
@@ -449,9 +449,11 @@ boost::optional<BlindSpotPolygons> BlindSpotModule::generateBlindSpotPolygons(
   lanelet::ConstLanelets blind_spot_lanelets;
   /* get lane ids until intersection */
   for (const auto & point : path.points) {
-    lane_ids.push_back(point.lane_ids.front());
-    if (point.lane_ids.front() == lane_id_) {
-      break;
+    for (const auto lane_id : point.lane_ids) {
```

Comment:
@takayuki5168 
you can use `std::set<int64_t> ` for in order not duplicate lane ids and please consider making 
this as planning_utils if same process is necessary for other scene module.